### DISCUSSION TITLE
Control of Wahoo Kickr Core resitance

### DISCUSSION THREAD

**Discussion by kruser123** [dvmarinoff/Auuki#164] [General]
Is it possible to use this app to control a Wahoo Kickr Core by only resistance (from 0 - 100)? The Wahoo app seems to have deprecated its resistance mode recently. Unfortunately, I train a handful of athletes and rely on this mode as the primary way of prescribing workouts. 

In addition, I would like to collect the following data:

- distance
- average speed
- average power (watts)
- time in minutes: seconds

Thanks in advance!

**Comment by dvmarinoff** [comment]
Well the short answer is it depends.  Since I don't have the Wahoo Kickr Core I can't run a test and tell, but on protocol level resistance is supported by this app on both WCPS (the old Wahoo bluetooth protocol) and FTMS (the new protocol that Wahoo updated to in 2021), the question is whether the Kickr Core supports it. If you are willing to help run some tests on the trainer itself and report back the logs from the browser I would be able to give a definitive answer.

**Comment by kruser123** [comment]
Thanks for getting back to me on this. I'm certainly willing to run some
tests on the trainer and report back the logs. Can you give me a list of
steps to take?
